### DISCUSSION TITLE
Toast: Is it just for breakfast

### DISCUSSION THREAD

**Discussion by tmknight** [VueTorrent/VueTorrent#747] [Polls]
Resolved: Toast notifications look snazzy, but do we need them?  

The argument: Toasts notifications tell us what we already know (you've logged in; you've pasted a URL, etc.).  They perform this community service at the expense of occupying precious screen real-estate (particularly on non-tablet/phablet mobile).  Yes, yes; I can hear you saying "but they're only there for few seconds" and "you know, you can click on them to make them go away quicker".  But why must I exert the energy to extend my finger or suffer to wait for their elderly shuffle off my screen?

So I put it to a vote: Toast or no Toast (or at least the option to disable them)?

PS: Hopefully you sense the humor in my tone and that this is not meant to start a flame war.  I seriously thank the developers and enjoy the work of this project a great deal.

**Comment by LeaderbotX400** [comment]
Well we are reworking the whole UI so we can change what the toasts do/say, any suggestions?

**Reply by tmknight** [reply]
The option to enable/disable toasts for those who prefer no toasts I think would off the most flexibility, so that those who like them may continue to see them...but that does add overhead to the developers.  

IMO, toasts should only be used when it is not obvious in any other way that an action was taken.  From my experience and POV with VT, I believe only the settings `save` actions meet that criteria and merit toast, all others have obvious indicators of an action taken (logon, paste URL, etc).

**Reply by LeaderbotX400** [reply]
Ok, thanks for the input. I am working on the vue 3 version so you won't see the changes until then (asumming WDaan agrees merges them).

**Reply by WDaan** [reply]
@tmknight I do agree with you, currently there are a couple of pop-ups that don't really provide any value and could be handled differently. We will keep this in mind while refactoring. Please don't exhaust your finger by tapping on the screen, trying to get rid of them early in the meantime.
